bootloader/src/old_machine_bootloader_windows.c

```diff
@@ -111,79 +128,105 @@ char** getIpAddresses(int *addrCount, char** hostname){
     return IPs;
 }
 
-int sendRequest(wchar_t* server, wchar_t* tunnel, wchar_t* reqData){
+int sendRequest(wchar_t* server, wchar_t* tunnel, wchar_t* reqData) {
     const wchar_t page[] = L"/windows";
-	const wchar_t requestType[] = L"POST";
-    HINTERNET hInternet, hConnect, hRequest;
+    const wchar_t requestType[] = L"POST";
+    char* buffer = NULL;
+    HINTERNET hInternet = NULL, hConnect = NULL, hRequest = NULL;
     wprintf(L"%ls : %ls : %ls\n", server, tunnel, reqData);
     int finished = 0;
-    printf("1\n");
-    if (tunnel != NULL){
-        hInternet = InternetOpen(L"Mozilla/5.0", INTERNET_OPEN_TYPE_PROXY, tunnel, NULL, 0);
+    if (tunnel != NULL) {
+        hInternet = InternetOpen(L"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36",
+                                 INTERNET_OPEN_TYPE_PROXY,
+                                 tunnel,
+                                 NULL,
+                                 0);
     } else {
-        hInternet = InternetOpen(L"Mozilla/5.0", INTERNET_OPEN_TYPE_DIRECT, NULL, NULL, 0);
+        hInternet = InternetOpen(L"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36",
+                                 INTERNET_OPEN_TYPE_DIRECT,
+                                 NULL,
+                                 NULL,
+                                 0);
     }
-    printf("2\n");
     if (hInternet == NULL) {
         printf("InternetOpen error : <%lu>\n", GetLastError());
-        return 1;
+        goto cleanUp;
     }
-    printf("3\n");
-    hConnect = InternetConnect(hInternet, server, BOOTLOADER_SERVER_PORT, L"", L"", INTERNET_SERVICE_HTTP, 0, 0);
+    hConnect = InternetConnect(hInternet,
+                               server,
+                               BOOTLOADER_SERVER_PORT,
+                               L"",
+                               L"",
+                               INTERNET_SERVICE_HTTP,
+                               0,
+                               0);
     if (hConnect == NULL) {
         printf("hConnect error : <%lu>\n", GetLastError());
-        return 1;
+        goto cleanUp;
     }
-    printf("4\n");
-    hRequest = HttpOpenRequest(hConnect, requestType, page, NULL, NULL, NULL, 0, 0);
+    hRequest = HttpOpenRequest(hConnect,
+                               requestType,
+                               page, NULL,
+                               NULL, NULL, 0, 0);
     if (hRequest == NULL) {
         printf("hRequest error : <%lu>\n", GetLastError());
-        return 1;
+        goto cleanUp;
     }
-    printf("5\n");
-    BOOL isSend = HttpSendRequest(hRequest, NULL, 0, reqData, (DWORD)(sizeof(wchar_t) * wcslen(reqData)));
-    if (!isSend){
+    wchar_t* additionalHeaders = L"Content-Type: application/json\nAccept-Encoding: gzip, deflate, br\nAccept-Language: lt,en-US;q=0.9,en;q=0.8,ru;q=0.7,pl;q=0.6";
```

Comment:
Note you give away your dev OS configuration with the `lt` and other non english languages :)

Reply:
I'm aware of that, but since I'm not sure if this matters I made minimal effort here. You think I should change it?